### DISCUSSION TITLE
Share article link or button on sidebar.

### DISCUSSION THREAD

**Discussion by felixdusengimana** [dailydotdev/daily#452] [Feature Requests]
Sharing articles on daily.dev seems kind complicated, maybe you can add link to homepage that allows users to add new articles on your blog.

**Comment by idoshamun** [comment]
Hi @phelixdusengimana,

What do you mean by "adding new articles on our blog"? Not sure I understand

**Reply by felixdusengimana** [reply]
Ok, I meat adding a link or something and when a user clicks it, he can easily share his article link or writes a markdown text and it is added to your blog.

**Reply by idoshamun** [reply]
We don't offer this option yet on the platform. But it's on our roadmap so stay tuned!

**Reply by felixdusengimana** [reply]
okey, thanks for the feedback!
